web/files/files.go

```diff
@@ -1485,6 +1485,7 @@ func FindFilesMango(c echo.Context) error {
 			v := v.(string)
 			if v == "path" {
 				includePath = true
+				fields = append(fields, "dir_id")
```

Comment:
It might be worth a small explanation comment? It makes sense that we needs the dirID to compute the path, but that's a bit surprising at first read.

Reply:
Added